### ISSUE TITLE
Apps Best Practice

### ISSUE THREAD

**Issue by OliverWilliamsDataManager** [medin-marine/horizon-scanning#24]
With the proliferation of apps devleoped by Scientists (e.g. ShinyR) it would be worth considering if MEDIN should produce some best practice guidance for data elements (hosting, metadata, transparency etc).  ICES have published some guidance from which we could borrow parts that are generic. [](https://doi.org/10.17895/ices.pub.22695898.v1)

**Comment by jensr** [comment]
Agreed - there is work from ICES which could be used as starting/comparative principles for taking on hosting roles for example.

**Comment by jensr** [comment]
https://10.17895/ices.pub.22695898
